students/M1lya1/apple_shop/scripts.js

```diff
@@ -0,0 +1,1324 @@
+        ram: null, // Гб
+        storage: null, // Гб
+        touchId: false,
+        faceId: false,
+        wireless: ['Bluetooth 5.0'],
+        camera: null,
+        audio: {
+            microphone: 'Three-mic array with directional beamforming',
+            speakers: 'Stereo speakers',
+        },
+        size: {
+            height: '5.3', // cm
+            width: '4.4', // cm
+            depth: '0.2', // cm
+            weight: '0.04', // Кг
+        },
+        os: null,
+        InTheBox: ['AirPods', 'Wireless Charging Case', 'Lightning to USB-A Cable', 'Documentation'],
+        orderInfo: {
+        	inStock: 23, // кол-во едениц товара в наличии
+        	reviews: 100, // процент положительных отзывов
+        }
+    },
+    {
+        id: 32,
+        category: 'airpods',
+        imgUrl: 'items/airpods_pro.jpg',
+        name: 'AirPods Pro',
+        display: null, // дюймы
+        color: ['White'],
+        price: 299, // доллары
+        chip: {
+            name: 'H!',
+            cores: null,
+        },
+        ram: null, // Гб
+        storage: null, // Гб
+        touchId: false,
+        faceId: false,
+        wireless: ['Bluetooth 5.0'],
+        camera: null,
+        audio: {
+            microphone: 'Three-mic array with directional beamforming',
+            speakers: 'Stereo speakers',
+        },
+        size: {
+            height: '4.5', // cm
+            width: '6.0', // cm
+            depth: '0.21', // cm
+            weight: '0.04', // Кг
+        },
+        os: null,
+        InTheBox: ['AirPods Pro', 'Wireless Charging Case', 'Lightning to USB-C Cable', 'Documentation', 'Silicone ear tips (three sizes)'],
+        orderInfo: {
+        	inStock: 29, // кол-во едениц товара в наличии
+        	reviews: 94, // процент положительных отзывов
+        }
+    },
+    {
+        id: 33,
+        category: 'airpods',
+        imgUrl: 'items/airpods-max.jpg',
+        name: 'AirPods Max',
+        display: null, // дюймы
+        color: ['White'],
+        price: 599, // доллары
+        chip: {
+            name: 'H!',
+            cores: null,
+        },
+        ram: null, // Гб
+        storage: null, // Гб
+        touchId: false,
+        faceId: false,
+        wireless: ['Bluetooth 5.0'],
+        camera: null,
+        audio: {
+            microphone: 'Three-mic array with directional beamforming',
+            speakers: 'Stereo speakers',
+        },
+        size: {
+            height: '18.7', // cm
+            width: '16.8', // cm
+            depth: '8.3', // cm
+            weight: '0.38', // Кг
+        },
+        os: null,
+        InTheBox: ['AirPods MAx', 'Smart Case', 'Lightning to USB-C Cable', 'Documentation'],
+        orderInfo: {
+        	inStock: 0, // кол-во едениц товара в наличии
+        	reviews: 5, // процент положительных отзывов
+        }
+    },
+];
+
+const acc = document.getElementsByClassName("accordion");
+let i;
+
+for (i = 0; i < acc.length; i++) {
```

Comment:
Consider to use `Array.from(acc)` to convert HTML collection to an array. And then use simple `forEach` cycle.